README.md

```diff
@@ -7,26 +7,26 @@
 
 <img style="text-align:center" src="https://s3.us-east-2.amazonaws.com/kepler-images/warrensbox/tfswitch/smallerlogo.png" alt="drawing" width="120" height="130"/>
 
-The `tfswitch` command line tool lets you switch between different versions of [Terraform](https://www.terraform.io/).  
-If you do not have a particular version of Terraform installed, `tfswitch` will download and verify the version you desire.  
-The installation is minimal and easy.  
+The `tfswitch` command line tool lets you switch between different versions of [Terraform](https://www.terraform.io/).
+If you do not have a particular version of Terraform installed, `tfswitch` will download and verify the version you desire.
+The installation is minimal and easy.
```

Comment:
Please put two trailing periods back — they are line breaks in markdown.

Reply:
Ah sorry, I have VS configured to auto remove trailing whitespace. This always catches me out as I avoid the 2 space trailing line breaks due to this break. I tend to use `<br>` instead if I need it based on [line breaks best practices](https://github.com/mattcone/markdown-guide/blob/master/_basic-syntax/line-breaks.md#line-break-best-practices)

Sorry, will put them back now.

Reply:
Fixed